app/controllers/job_templates_controller.rb

```diff
@@ -24,6 +24,26 @@ def preview
     end
   end
 
+  def import
+    contents = params[:imported_template][:template].respond_to?(:read) ? params[:imported_template][:template].read : nil
```

Comment:
maybe a bit safer and more readable way
`contents = params.fetch(:imported_template, {}).fetch(:template, nil).try(:ready)`